### DISCUSSION TITLE
Python modules in different directory from Makefile?

### DISCUSSION THREAD

**Discussion by dalleggio** [cocotb/cocotb#2928] [Q&A] [answered]
* the cocotb version used: 1.6.2
* the operating system and version: Kubuntu (64 bit)
* the simulator and version: Models (32 bit),
* the Python version, and where it's coming from Minconda, Python 3.7.1

Is there a way to have the python code for modules in a different directory from where the Makefile is?

Thanks
Anthony

**Comment by ktbarrett** [comment, marked as answer]
Set [`PYTHONPATH`](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH).

**Comment by matthuszagh** [comment]
In case it's helpful to anyone, https://github.com/cocotb/cocotb/blob/6a3220c5ed6c05a48648593479438bdfa5c91648/examples/adder/tests/Makefile#L34 is an example of setting `PYTHONPATH`.
